src/MassTransit/Configuration/SerializerConfigurationExtensions.cs

```diff
@@ -25,6 +26,35 @@ public static void UseJsonSerializer(this IReceiveEndpointConfigurator configura
             configurator.SetMessageSerializer(() => new JsonMessageSerializer());
         }
 
+        /// <summary>
+        /// Serialize messages using the System.Text.Json serializer
+        /// </summary>
+        /// <param name="configurator"></param>
+        /// <param name="options"></param>
+        public static void UseMicrosoftJsonSerializer(this IBusFactoryConfigurator configurator,
+            Func<JsonSerializerOptions, JsonSerializerOptions> options = null)
+        {
+            if (options != null)
+            {
+                SystemTextJsonConfiguration.Options = options(SystemTextJsonConfiguration.Options);
+            }
+
+            configurator.SetMessageSerializer(() => new SystemTextJsonMessageSerializer());
+            configurator.AddMessageDeserializer(SystemTextJsonMessageSerializer.JsonContentType,
+                () => new SystemTextJsonMessageDeserializer());
+        }
+
+        /// <summary>
+        /// Serialize messages using the System.Text.Json serializer
+        /// </summary>
+        /// <param name="configurator"></param>
+        public static void UseMicrosoftJsonSerializer(this IReceiveEndpointConfigurator configurator)

```

Comment:
It's actually `System.Text.Json`, so for consistency let's go with `UseSystemTextJsonSerializer` okay?

Reply:
done.